doc/release/3.3.0.rst

```diff
@@ -0,0 +1,375 @@
+            log.info('roles_cfg.my_role.foo: ' .. config.foo)
+        end,
+    }
+
+.. _3-3-alert-api:
+
+API for raising alerts
+~~~~~~~~~~~~~~~~~~~~~~
+
+Now developers can raise their own alerts from their application or application roles.
+For this purpose, a new API is introduced into the ``config`` module.
+
+The ``config:new_alerts_namespace()`` function creates a new
+*alerts namespace* -- a named container for user-defined alerts:
+
+.. code-block:: lua
+
+    local config = require('config')
+    local alerts = config:new_alerts_namespace('my_alerts')
+
+Alerts namespaces provides methods for managing alerts within them. All user-defined
+alerts raised in all namespaces are shown in ``box.info.config.alerts``.
+
+To raise an alert, use the namespace methods ``add`` or ``set``:
+The difference between them is that ``set()`` accepts a key to use as an alert
+identifier for further operations. An alert must be a table; it's ``message`` field is logged.
+
+.. code-block:: lua
+
+    -- Raise a new alert.
+    alerts:add({
+        message = 'Test alert',
+        my_field = 'my_value',
+    })
+
+    -- Raise a new alert with a key.
+    alerts:set("my_alert", {
+        message = 'Test alert',
+        my_field = 'my_value',
+    })
+
+You can discard alerts individually by keys using the ``unset()`` method, or
+all at once using ``clear()``:
+
+.. code-block:: lua
+
+    alerts:unset("my_alert")
+    alerts:clear()
+
+.. _3-3-administration-and-maintenance:
+
+Administration and maintenance
+------------------------------
+
+.. _3-3-upgrade-ddl:
+
+DDL before upgrade
+~~~~~~~~~~~~~~~~~~~
+
+* Апгрейд лишился ограничений на DDL по сравнению с версиями 3.0, 3.1, 3.2.
+Это позволяет, например, выдать права на функции до вызова box.schema.upgrade()..
+
+Tarantool now allows DDL operations before calling ``box.schema.upgrade()`` during
+an upgrade from an earlier version. For example, you can
+
+.. _3-3-isolated-instances:
+
+Isolated instances
+~~~~~~~~~~~~~~~~~~
+
+A new instance-level configuration option ``isolated`` puts an instance into the
+*isolated* mode.
+
+.. code-block:: yaml
+
+    groups:
+      group-001:
+        replicasets:
+          replicaset-001:
+            instances:
+              instance-001: {}
+              instance-002: {}
+              instance-003:
+                isolated: true
+
+In this mode, an instance doesn't accept updates from other members of its replicaset
+and other iproto requests. It also doesn't perform background data modifications
+and remains in read-only mode.
+
+Use the isolated mode to temporarily isolate instances for maintenance, debugging,
+or other actions that should not affect other cluster instances.
+
+.. _3-3-replica-unregister:
+
+Automatic expulsion configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new configuration section ``replication.autoexpel`` allows to automatically expel
+instances after they are removed from the YAML configuration.
```

Comment:
It is OK to use `expulse` and `expel` both? I would attempt to use one term for one feature.

Reply:
Most dictionaries state that `expulsion` is the noun form of `expel` (a surprise for me), so it's the same term.

Reply:
Hm, OK so.